exp/orderbook/graph_test.go

```diff
@@ -320,6 +321,15 @@ func assertPathEquals(t *testing.T, a, b []Path) {
 	}
 }
 
+func findByAsset(edges edgeSet, asset string) Venues {
+	for i, key := range edges.keys {
+		if key == asset {
+			return edges.values[i]
+		}
+	}
+	return Venues{}
+}
```

Comment:
This can use the other helper `findKey`, right?

```suggestion
func findByAsset(edges edgeSet, asset string) Venues {
	i := findKey(edges.keys, asset)
	if i >= 0 {
		return edges.values[i]
	}
	return Venues{}
}
```